packages/@glimmer/compiler/lib/template-compiler.ts

```diff
@@ -178,18 +178,13 @@ export default class TemplateCompiler implements Processor<InputOps> {
         this.opcode(['attrSplat'], action);
       } else if (isStatic && !isComponent) {
         this.opcode(['staticAttr', [name, namespace]], action);
-      } else if (isTrusting) {
+      } else if (isStatic || isTrusting) {
```

Comment:
I don't quite think this is the same as before. Specifically, I don't think that we would inherently trust static attributes and that instead when `isStatic && isComponent` we should use `componentAttr` (vs `trustingComponentAttr`).

Reply:
Problem is that static element with modifier is treated as component (template-compiler.ts:102), so ```isStatic && isComponent``` means also static element + static styles + element modifier. So glimmer will complain about potential XSS in styles (test will fail, because static styles will not be trusted)

I can change condition, so it will be specific for styles only 
```if ((isStatic && name === 'style') || isTrusting)```
so other static attributes with isComponent will use ```componentAttr```, and static styles will use "trustingComponentAttr".

What do you think about it ?